pkg/ui/react-app/src/pages/notFound/NotFound.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC } from 'react';
```

Comment:
Can you move this component to `src/thanos/pages`? All of our custom pages are there.

Reply:
Ofcourse !
Is everything else fine ?

Reply:
Yep, everything else seems to be alright. I'll test it out locally first and then let's get this merged.